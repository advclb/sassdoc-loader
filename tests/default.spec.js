const compiler = require("./compiler");

test("Output match snapshot", async function () {
  const stats = await compiler("source.scss");
  const output = stats.toJson().modules[0].source;
  expect(output).toMatchSnapshot();
});
