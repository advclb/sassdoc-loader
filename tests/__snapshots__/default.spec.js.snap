// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inserts name and outputs JavaScript 1`] = `"export default [{\\"description\\":\\"This is a function\\\\nThe function does nothing\\\\n\\",\\"commentRange\\":{\\"start\\":11,\\"end\\":17},\\"context\\":{\\"type\\":\\"function\\",\\"name\\":\\"message\\",\\"code\\":\\"\\\\n  @return null;\\\\n\\",\\"line\\":{\\"start\\":18,\\"end\\":20}},\\"access\\":\\"private\\",\\"group\\":[\\"messages\\"],\\"parameter\\":[{\\"type\\":\\"String\\",\\"name\\":\\"theme\\",\\"description\\":\\"Theme name\\"},{\\"type\\":\\"String\\",\\"name\\":\\"key\\",\\"description\\":\\"Key name\\"}],\\"return\\":{\\"type\\":\\"String\\"},\\"file\\":{\\"path\\":\\"source.scss\\",\\"name\\":\\"source.scss\\"}},{\\"description\\":\\"This is a mixin\\\\nThe mixin does nothing\\\\n\\",\\"commentRange\\":{\\"start\\":3,\\"end\\":7},\\"context\\":{\\"type\\":\\"mixin\\",\\"name\\":\\"set-minimum-level\\",\\"code\\":\\"\\\\n\\",\\"line\\":{\\"start\\":8,\\"end\\":9}},\\"access\\":\\"public\\",\\"parameter\\":[{\\"type\\":\\"String\\",\\"name\\":\\"level\\"}],\\"output\\":\\"Nothing\\",\\"group\\":[\\"undefined\\"],\\"file\\":{\\"path\\":\\"source.scss\\",\\"name\\":\\"source.scss\\"}}];"`;

exports[`Output match snapshot 1`] = `"export default [{\\"description\\":\\"This is a function\\\\nThe function does nothing\\\\n\\",\\"commentRange\\":{\\"start\\":11,\\"end\\":17},\\"context\\":{\\"type\\":\\"function\\",\\"name\\":\\"message\\",\\"code\\":\\"\\\\n  @return null;\\\\n\\",\\"line\\":{\\"start\\":18,\\"end\\":20}},\\"access\\":\\"private\\",\\"group\\":[\\"messages\\"],\\"parameter\\":[{\\"type\\":\\"String\\",\\"name\\":\\"theme\\",\\"description\\":\\"Theme name\\"},{\\"type\\":\\"String\\",\\"name\\":\\"key\\",\\"description\\":\\"Key name\\"}],\\"return\\":{\\"type\\":\\"String\\"},\\"file\\":{\\"path\\":\\"source.scss\\",\\"name\\":\\"source.scss\\"}},{\\"description\\":\\"This is a mixin\\\\nThe mixin does nothing\\\\n\\",\\"commentRange\\":{\\"start\\":3,\\"end\\":7},\\"context\\":{\\"type\\":\\"mixin\\",\\"name\\":\\"set-minimum-level\\",\\"code\\":\\"\\\\n\\",\\"line\\":{\\"start\\":8,\\"end\\":9}},\\"access\\":\\"public\\",\\"parameter\\":[{\\"type\\":\\"String\\",\\"name\\":\\"level\\"}],\\"output\\":\\"Nothing\\",\\"group\\":[\\"undefined\\"],\\"file\\":{\\"path\\":\\"source.scss\\",\\"name\\":\\"source.scss\\"}}];"`;
